tests/scripts/check_files.py

```diff
@@ -370,7 +370,7 @@ class LicenseIssueTracker(LineIssueTracker):
     LICENSE_EXEMPTION_RE_LIST = [
         # Third-party code, other than whitelisted third-party modules,
         # may be under a different license.
-        r'3rdparty/(?!(p256-m)/.*)',
+        r'tf-psa-crypto/drivers/(?!(builtin|p256-m)/.*)',
```

Comment:
Since we no longer have a dedicated `3rdparty` directory, it'd be nice to have a default of "this cannot have a different license" (i.e. safe by default) in case we add other official drivers in future.

I believe this could be achieved like so:
```suggestion
        r'tf-psa-crypto/drivers/(?=(everest)/.*)',
```

Would also need to change the description to something like:
```python
# Exempt third-party drivers which may be under a different license
```

Reply:
Yes good idea, thanks.